Source/Core/VideoCommon/BPMemory.h

```diff
@@ -491,6 +491,81 @@ struct fmt::formatter<TevStageCombiner::ColorCombiner>
   template <typename FormatContext>
   auto format(const TevStageCombiner::ColorCombiner& cc, FormatContext& ctx)
   {
+    auto out = ctx.out();
+    if (cc.bias == TevBias::Compare)
+    {
+      // TODO
```

Comment:
This TODO was left blank

Reply:
I don't think there's really a good way of formatting it when in compare mode; compare mode doesn't have that much of an equation, but what it does have is awkward to format.  I'll just get rid of the TODO comment.